kolibri/core/assets/src/views/SideNav.vue

```diff
@@ -63,6 +63,26 @@
           </CoreMenu>
 
           <div class="side-nav-scrollable-area-footer" :style="{ color: $themeTokens.annotation }">
+            <!-- custom branded footer logo + text -->
+            <img
+              v-if="$theme.sideNav.brandedFooter.logo"
+              class="side-nav-scrollable-area-footer-logo"
+              :src="$theme.sideNav.brandedFooter.logo.src"
+              :alt="$theme.sideNav.brandedFooter.logo.alt"
+              :style="$theme.sideNav.brandedFooter.logo.style"
+            >
+            <div
+              v-if="$theme.sideNav.brandedFooter.logo"
```

Comment:
Should this actually be

```
v-if="$theme.sideNav.brandedFooter.textLines && $theme.sideNav.brandedFooter.textLines.length"
```

or similar?

Reply:
this I wasn't sure about, whether we were ok just displaying some floating text without an image.

Reply:
I don't know from a design perspective, but from a code perspective this way is "more correct".

As an aside, you can use templates to do a single conditional rather than repeating it in multiple elements:

```html
<template v-if="someCondition">
  <img src="..." />
  <div>...</div>
</template>
```

rather than

```html
<img v-if="someCondition" src="..." />
<div v-if="someCondition">...</div>
```